components/front/prizePackages.tsx

```diff
@@ -0,0 +1,51 @@
+import styles from './prizePackages.module.css';
+import Fade from 'react-reveal/Fade';
+interface Prize {
+	name: string;
+	color: string;
+}
+
+const PrizeInfo: Prize[] = [
+	{
+		name: 'outdoors',
+		color: '#A4D5B2',
+	},
+	{
+		name: 'lazyOutdoors',
+		color: '#7968AE',
+	},
+	{
+		name: 'tunes',
+		color: '#D70E65',
+	},
+	{
+		name: 'netflix',
+		color: '#AA2208',
+	},
+	{
+		name: 'beach',
+		color: '#FFD84C',
+	},
+];
+const Packages = () => {
+	return (
+		<div className={styles.row}>
+			<div className={styles.prizes}>
+				<Fade>
+					{PrizeInfo.map(({ name, color }) => {
+						return (
+							<div className={styles.column} key={name}>
+								<div className={styles.card}>
+									&nbsp;
+									<img alt="" src={`images/prizes/${name}.png`} />
```

Comment:
We should use the `alt` tag here for accessibility. Can you add a `description` field to each Prize and copy over the text from the image and use that as the alt?

Reply:
yeah cool good point my bad